x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/right/components/threat_intelligence_overview.tsx

```diff
@@ -88,6 +88,17 @@ export const ThreatIntelligenceOverview: FC = () => {
         title: TITLE,
         link,
         iconType: !isPreviewMode ? 'arrowStart' : undefined,
+        headerContent: (
+          <EuiIconTip
+            content={
+              <FormattedMessage
+                id="xpack.securitySolution.flyout.right.threatintelligence.time"
+                defaultMessage="The threat intelligence is calculated from a specific time interval. To change that interval, go to the Threat intelligence tab in the flyout expanded view and use the date picker."
```

Comment:
would be great to have docs review. I wonder if `flyout expanded view` is clear to users, alternatively they can click on the threat intelligence title to open it directly
<img width="567" height="145" alt="image" src="https://github.com/user-attachments/assets/a73cbd0f-36f3-47af-b1e5-61bcf5ed352d" />